bokeh/plotting/helpers.py

```diff
@@ -724,6 +724,10 @@ def func(self, **kwargs):
         renderer_kws = _pop_renderer_args(kwargs)
         source = renderer_kws['data_source']
 
+        # Assign global_alpha from alpha if glyph type is an image
+        if 'alpha' in kwargs and 'Image' in glyphclass.__view_model__:
+            kwargs['global_alpha'] = kwargs['alpha']
+
```

Comment:
I think I'd be happier with this with explicit instance checks. Alternatively a generic way to do this would be to delegate to the glyphclasses to handle directly somehow but that does not need to happen right now. 
```

Reply:
OK, I'll change to explicit instance checks. The __view_model__ approach seemed neat but I guess it's not foolproof against 'Image' being in an unsuitable view model in the future.